src/org/zaproxy/zap/extension/autoupdate/ExtensionAutoUpdate.java

```diff
@@ -1558,7 +1578,102 @@ public boolean supportsDb(String type) {
         		"-addonlist               " + Constant.messages.getString("cfu.cmdline.list.help"));
         return arguments;
     }
+    
+    public void installAddOns(Vector<String> addons) {
```

Comment:
Does it really need to be a `Vector`? (`List` should work for both cases).
Should be synchronised?